solr/solrj/src/java/org/apache/solr/common/util/Utils.java

```diff
@@ -909,6 +909,15 @@ public static void reflectWrite(MapWriter.EntryWriter ew, Object o) {
    * @return a serializable version of the object
    */
   public static Object getReflectWriter(Object o) {
+    // Defensive: Avoid reflective serialization for Lucene Point types (e.g., LongPoint)
+    // which are not meant to be serialized this way and can cause IllegalAccessException.
+    String lucenePointPkg = "org.apache.lucene.document.";
+    if (o != null
+        && o.getClass().getName().startsWith(lucenePointPkg)
```

Comment:
This feel like a hack to me.  Perhaps it's the best choice but hard for me to say at this time before understanding how this is invoked.  Maybe there's a solution preventing a Point field from getting here in the first place.  A test would help!